### DISCUSSION TITLE
[@types/ramda] 0.28.18 remove legacy functions

### DISCUSSION THREAD

**Discussion by megakoresh** [DefinitelyTyped/DefinitelyTyped#62943] [Issues with a @types package]
There are some functions which the types still export but which are not anymore in the main library. Namely:

```
R.merge -> replaced with R.mergeDeepLeft
R.contains -> replaced with R.includes
```

These need to be removed from the types so it's actually in sync with the main library. I guess there might be others, haven't had time to test all of them yet.

[a comment by an automated account is not shown]

**Comment by Nicholaiii** [comment]
Feel free to PR it

**Reply by megakoresh** [reply]
I will once I have time to set up all the infrastructure needed to contribute here and familiarize myself with the process, but for someone who already has all of this set up it's like a 5 minute job, so I figured I'd open an issue about it first.

**Comment by asbjorn-brevio** [comment]
I think #64050 should fix this. Sorry for reporting duplicate issue (#64001). I didn't know that GH Discussions exists 😄
